README.md

```diff
@@ -91,7 +91,4 @@ Join us [here](https://discord.gg/nNtVfKddDD).
 ## Streaming
 
 I (YK) stream almost every day showing my progress on this project on Twitch [here](https://twitch.tv/ykdojo).
-
-## Resources
-
-You can find YK's original tweet [here](https://twitter.com/ykdojo/status/1557611357251350528).
+##
```

Comment:
remove this line

Reply:
I have done that. 
It's still rejecting my PR.
I don't know what's going on

Reply:
I have done that. 
It's still rejecting my PR.
I don't know what's going on